src/ui/units/collections/components/CollectionActions/components/SharedEntryNotice/SharedEntryNotice.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import block from 'bem-cn-lite';
+
+import {getSharedEntryMockText} from '../../../helpers';
+
+import './SharedEntryNotice.scss';
+
+const b = block('dl-shared-entry-actions-notice');
+
+export const SharedEntryNotice = () => {
```

Comment:
Looks like Alert https://preview.gravity-ui.com/uikit/?path=/story/components-feedback-alert--theme

Reply:
What is it that the alert does not allow you to do at the moment?